tg_global_gsql.yaml

```diff
@@ -0,0 +1,21 @@
+---
+# @           : this file
+# {x}         : the directory of file x
+# &x          : any file which matches pattern x
+# ...         : any depth of arbitrary directories
```

Comment:
suggest to use ** instead ..., as ** is used in [glob](https://docs.python.org/3/library/glob.html)

Reply:
yes